src/ha_addon_sunsynk_multi/options.py

```diff
@@ -52,24 +53,24 @@ class Options:
 
     _LISTS = {"inverters": InverterOptions, "schedules": Schedule}
 
-    mqtt_host: str = ""
-    mqtt_port: int = 0
-    mqtt_username: str = ""
-    mqtt_password: str = ""
-    number_entity_mode: str = "auto"
-    prog_time_interval: int = 15
+    mqtt_host: str = os.getenv("MQTT_HOST", "")
+    mqtt_port: int = int(os.getenv("MQTT_PORT", 0))
```

Comment:
pylint seems to expect strings as the default arg

```suggestion
    mqtt_port: int = int(os.getenv("MQTT_PORT", "0"))
```